public/app/plugins/datasource/cloudwatch/datasource.js

```diff
@@ -191,7 +185,20 @@ function (angular, _) {
           });
         }
 
-        return this.getDimensionValues(region, namespace, metricName, dimensions);
+        return this.getDimensionValues(region, namespace, metricName, dimensions).then(function(result) {
+          return _.map(result, function(dimensions) {
+            var values = _.chain(dimensions)
+            .sortBy(function(dimension) {
+              return dimension.Name;
+            })
+            .map(function(dimension) {
+              return dimension.Name + '=' + dimension.Value;
```

Comment:
when and how will that be used? the target.dimensions is an object, can only use templating for the key or for the key value, not both at the same time (for a single variable)